### PR TITLE
Incorrect responses logged to CRM

### PR DESCRIPTION
**Pull request by imsickofmaps**
When filling in M&L wrong answer gets logged

### CODE REVIEW

**Comment by imsickofmaps** [comment]
@smn ready for review

File: js_sandbox/lib/go-nike-ghr.js [hunk 1]
```diff
@@ -884,19 +854,34 @@ function GoNikeGHR() {
...
+                    item++;
                     var question = quiz.questions[question_name];
                     var question_state_name = quiz_name + "_" + question_name;
-
-                    // construct a function using make_question_state()
+                    // construct a function using make_mandl_question_state()
                     // to prevent getting a wrongly scoped 'question'
-                    self.add_creator_unless_exists(question_state_name,
-                        self.make_question_state(quiz_name, question));
+                    if (item !== 1){
```
Comment (smn) [inline js_sandbox/lib/go-nike-ghr.js:866]: Can you explain why this is necessary?

Comment (imsickofmaps) [inline js_sandbox/lib/go-nike-ghr.js:866]: The first item we're not ready to build the questions yet, we need a pairing of previous_question_state_name and question_state_name to build it

**Comment by smn** [comment]
:+1:

**Pull request merged**